files/en-us/web/api/keyboardevent/code/code_values/index.html

```diff
@@ -2611,13 +2521,13 @@ <h3 id="Code_values_on_Linux_X11_When_scancode_is_available">Code values on Linu
   </tr>
   <tr>
    <th scope="row"><code>0x00E1</code></th>
-   <td style="background-color: rgb(255, 255, 204);"><code>"BrowserSearch"</code></td>
-   <td style="background-color: rgb(255, 255, 204);"><code>""</code></td>
+   <td><code>"BrowserSearch"</code></td>
+   <td><code>""</code> (❌ Missing)</td>
   </tr>
  </tbody>
 </table>
 
-<h3 id="Code_values_on_Android_and_Firefox_OS_When_scancode_is_available">Code values on Android and Firefox OS (When scancode is available)</h3>
+<h2 id="Code_values_on_Android_and_Firefox_OS_When_scancode_is_available">Code values on Android and Firefox OS (When scancode is available)</h2>
```

Comment:
...and here, and remove the mention of Firefox OS:

```suggestion
<h2 id="Code_values_on_Firefox_for_Android">Code values on Firefox for Android</h2>
```